### ISSUE TITLE
Research Document Viewers

### ISSUE THREAD

**Issue by khandelwal** [18F/foia-hub#455]
We'll need to display the contents of PDF documents for example, likely inline on our document landing page.

Research the various methods to do so, and pick one that works well. Here's an example of the one the FBI's vault uses: http://vault.fbi.gov/joe-paterno/joe-paterno-part-01-of-01/view

**Comment by khandelwal** [comment]
This is definitely a task that friendly folks that are not on the core team can easily participate in - so I'm adding the 'help_wanted' tag.

**Comment by jackiekazil** [comment]
Document cloud's view is nice, but it comes with a lot more than I think this project needs. 
So, I would recommend pdf.js -- https://github.com/mozilla/pdf.js/

**Comment by konklone** [comment]
FWIW, I did some work trying to integrate pdf.js into [Scout](https://scout.sunlightfoundation.com), and it's heavy in its own way. It's highly optimized for Mozilla's needs as a full in-browser PDF viewer, with all the bells and whistles. Hard to extract just the viewing logic.

**Comment by acdha** [comment]
Yes, it was generally just with the expectation of large documents. Acrobat, Chrome/pdfium, Safari, etc. are heavily tested and optimized native code. pdf.js has improved rapidly but it's still noticeably slower for large documents.

The other issue is features: if you need any features other than the core PDF spec at the level somewhere around [PDF/A](http://en.wikipedia.org/wiki/PDF/A) your options will quickly shrink to the system frameworks or just Acrobat. Hopefully that last won't be an issue as browser support for plugins is rapidly being deprecated.

**Comment by ccostino** [comment]
Thanks for answering, @acdha!  I didn't get a chance to sit down until just
now and would not have been able to provide all of the detailed info you
did (I do recall just relying on the native PDF support found within the
browser itself, or the Adobe Reader plugin if it was installed).  So thank
you, and @jackiekazil, what @acdha said. :-)

**Comment by jackiekazil** [comment]
Ty for chiming in!
